src/module/iotjs_module_gpio.c

```diff
@@ -143,6 +143,7 @@ void iotjs_gpio_after_worker(uv_work_t* work_req, int status) {
           iotjs_jargs_append_error(&jargs, "GPIO Close Error");
         } else {
           iotjs_jargs_append_null(&jargs);
+          iotjs_jargs_append_bool(&jargs, req_data->value);
```

Comment:
AFAIK: In case of close the `value` does not hold any meaningful information, as it is not set by any of the platform/board code part.